java/src/com/ibm/streamsx/topology/generator/spl/GraphUtilities.java

```diff
@@ -135,6 +137,51 @@
         uniqueParents.addAll(parents);
         return uniqueParents;
     }
+    
+    /**
+     * Copies operator, giving it a new name. Also renames input/output ports,
+     * and clears the input/output connections array.
+     * @param op
+     * @param name
+     */
+    static JSONObject copyOperatorNewName(JSONObject op, String name){
+	try{
+	    op = OrderedJSONObject.parse(op.serialize());
+	}
+	catch(Exception e){
+	    e.printStackTrace();
+	    System.exit(-1);
```

Comment:
Don't call System.exit(), wrap the Exception in a RuntimeException and throw it.